README.md

```diff
@@ -1,26 +1,34 @@
-Sangbogen
-========
+# Sangbogen
 
-This is the current songbook in the F-Klub
+This is the current songbook of F-klubben.
+
+## Download
 
-Download
--------------
 The latest songbook can be downloaded from [here](https://github.com/f-klubben/sangbog/releases/latest/download/sangbog.pdf) in booklet format ready for printing. If you require the songbook in a non-booklet format, you can follow the build instructions below (and use the `pdf` target for`make`).
 
-Building from source
--------------
+## Building from source
+
 For Debian-based systems (in other distributions package names may vary):
+
 1. Fetch the source code
-  ```sh
-  git clone https://github.com/f-klubben/sangbog.git
-  ```
+
+```sh
+git clone https://github.com/f-klubben/sangbog.git
+```
+
 2. Install prerequisites:
-  ```sh
-  sudo apt install texlive-latex-extra psutils texlive-lang-european ghostscript
-  ```
+
+```sh
+sudo apt install texlive-latex-extra psutils texlive-lang-european ghostscript
+```
+
 3. Build the sangbog
-  `make booklet` for booklet format, `make pdf` for non-booklet (continuous) format
+   `make booklet` for booklet format, `make pdf` for non-booklet (continuous) format
 
+## Adding new songs
+
+
+=======
```

Comment:
![billede](https://github.com/f-klubben/sangbog/assets/17174333/bc44bfa2-5f77-4a84-bc47-c0af8f18cc86)
I'm not sure how this is supposed to look

Reply:
Ooops, Imma fix it.